modules/kafka/kafka-dest-driver.c

```diff
@@ -173,7 +281,8 @@ _kafka_delivery_report_cb(rd_kafka_t *rk,
   else
     {
       msg_debug("kafka: delivery report successful",
-                evt_tag_str("topic", self->topic_name),
+                evt_tag_str("topic", self->topic_name->template),
+                evt_tag_str("fallback_topic", self->fallback_topic_name ? self->fallback_topic_name : "NULL"),
```

Comment:
evt_tag_str() handles NULLs, so the check can be avoided (everywhere).

Reply:
Done